gradle/versions.gradle

```diff
@@ -13,7 +13,7 @@
 
 dependencyManagement {
   dependencies {
-    dependency 'com.fasterxml.jackson.core:jackson-databind:2.13.1'
+    dependency 'com.fasterxml.jackson.core:jackson-databind:2.13.2.2'
```

Comment:
Can use a dependency set and also update the other jackson library versions as well i.e. the dataformat-yml package